src/components/List/ListItem/ListItem.story.jsx

```diff
@@ -185,4 +194,13 @@ storiesOf('Watson IoT Experimental/ListItem', module)
         ]}
       />
     </div>
+  ))
+  .add('with Tags', () => (
+    <div style={{ width: 400 }}>
+      <ListItem
+        id="list-item"
+        value={text('value', 'List Item')}
+        tags={[{ type: 'blue', content: 'default' }, { type: 'red', content: 'warning' }]}
```

Comment:
If we modify the proptype to be an array of nodes, this could be something like:

```suggestion
        tags={[<Tag type='blue' title='descriptor'>default</Tag>, <Tag type='red' disabled>disabled tag</Tag>]}
```